packages/mutation-testing-elements/src/components/mutation-test-report-app/theme.scss

```diff
@@ -58,35 +59,91 @@
     --bs-btn-#{$color}-color: #{color-yiq($value)};
     --bs-btn-#{$color}-hover-color: #{color-yiq(darken($value, 7.5%))};
     --bs-#{$color}-bg: #{$value};
+    --bs-#{$color}-bg-light: #{rgba($value, 0.25)};
   }
 }
 
 // Prismjs theming (code highlighting) =====================================
 :host(:not([theme='dark'])) {
-  --prism-maintext: black;
-  --prism-background: #f5f2f0;
-  --prism-text: #999;
-  --prism-keywords: #07a;
-  --prism-punctuation: #999;
-  --prism-functions: #dd4a68;
-  --prism-strings: #690;
-  --prism-namespace: slategray;
-  --prism-names: #905;
-  --prism-symbol: #dd4a68;
-  --prism-url: #67cdcc;
+  --prism-maintext: #393a34;
+  --prism-background: #f6f8fa;
+  --prism-border: #ddd;
+
+  // from https://github.com/FormidableLabs/prism-react-renderer/blob/master/src/themes/github.js
+  --prism-operator: var(--prism-maintext);
+  --prism-punctuation: var(--prism-maintext);
+
+  --prism-atrule: #00a4db;
+  --prism-attr: var(--prism-atrule);
+  --prism-selector: var(--prism-atrule);
+
+  --prism-cdata: #999988;
+  --prism-comment: var(--prism-cdata);
+  --prism-doctype: var(--prism-cdata);
+  --prism-prolog: var(--prism-cdata);
+
+  --prism-deleted: #6f42c1;
+  --prism-function: var(--prism-deleted);
+  --prism-tag: var(--prism-deleted);
+
+  --prism-boolean: #36acaa;
+  --prism-constant: var(--prism-boolean);
+  --prism-entity: var(--prism-boolean);
+  --prism-inserted: var(--prism-boolean);
+  --prism-number: var(--prism-boolean);
+  --prism-property: var(--prism-boolean);
+  --prism-regex: var(--prism-boolean);
+  --prism-symbol: var(--prism-boolean);
+  --prism-url: var(--prism-boolean);
+  --prism-variable: var(--prism-boolean);
+
+  --prism-attr: #e3116c;
+  --prism-string: var(--prism-attr);
+
+  --prism-keyword: #00009f;
+  --prism-selector: var(--prism-keyword);
+  --prism-tag: var(--prism-keyword);
 }
 :host([theme='dark']) {
-  --prism-maintext: #ccc;
+  // From https://atelierbram.github.io/syntax-highlighting/prism/demo/assets/css/prism-base16-eighties.dark.css
```

Comment:
So in the end you didn't use github dark theme?

Reply:
No, this is closer to [this theme](https://github.com/chriskempson/tomorrow-theme) which is what the original dark theme was based on, but discontinued